Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -79,7 +80,9 @@ public override void Update(float frameTime)
                     continue;
                 }
 
-                _material.SpawnMultipleFromMaterial(reclaimer.CurrentExpectedYield, "Biomass", Transform(uid).Coordinates);
+                var actualYield = (int) (reclaimer.CurrentExpectedYield); // can only have integer biomass
+                reclaimer.CurrentExpectedYield = reclaimer.CurrentExpectedYield - actualYield; // store non-integer leftovers
+                _material.SpawnMultipleFromMaterial(actualYield, "Biomass", Transform(uid).Coordinates);
```

Comment:
make Biomass a constant string  field with the ValidatePrototypeId attribute in the system